python/Homework.py

```diff
@@ -0,0 +1,210 @@
+import numpy as np
+import pandas as pd
+import plotly.express as px
+from sklearn import tree
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import StandardScaler
+
+# 1 Load the above data into a Pandas DataFrame
+heads = (
+    "sepal length in cm",
+    "sepal width in cm",
+    "petal length in cm",
+    "petal width in cm",
+    "class",
+)
+Iris_df = pd.read_csv(
+    "https://archive.ics.uci.edu/ml/machine-learning-databases/iris/iris.data",
+    names=heads,
+)
+Iris_df1 = pd.read_csv(
+    "https://archive.ics.uci.edu/ml/machine-learning-databases/iris/iris.data"
+)
+
+nd_array_1 = np.array(Iris_df1)[:, 0:4]
```

Comment:
slide [:, 0:4] could be replaced by [:, :4] for short

Reply:
Thanks!